lib/screens/settings/activity_feed.dart

```diff
@@ -121,9 +125,16 @@ class _ActivityFeedScreenState extends State<ActivityFeedScreen> {
                                         });
                                   });
                             })),
+                    validator: (value) {
+                      if (value == null || Uri.tryParse(value) != null) {
```

Comment:
```suggestion
                      if (value == null || Uri.tryParse(value) == null) {
```

Reply:
Actually the first null check was also wrong, thanks for catching that.